homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -380,7 +380,7 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
         host = config_entry.data[CONF_HOST]
         token = config_entry.data[CONF_TOKEN]
         model = config_entry.data[CONF_MODEL]
-        device = hass.data[DOMAIN][config_entry.entry_id].get(KEY_DEVICE)
+        device = hass.data[DOMAIN].get(config_entry.entry_id, {}).get(KEY_DEVICE)
```

Comment:
I'd prefer if we make two cases explicitly where we set `device` so that it doesn't look as `device` can be `None`. Mypy would complain about this too if we would have typing in place.